frontend/src/metabase/collections/components/CollectionHeader/CollectionCaption.tsx

```diff
@@ -92,6 +92,10 @@ const CollectionCaptionIcon = ({ collection }: { collection: Collection }) => {
     );
   }
 
+  if (PLUGIN_COLLECTIONS.isSyncedCollection(collection)) {
```

Comment:
I find it weird that synced collections and instance analytics collection checks are done so differently